client/app/routes/index.coffee

```diff
@@ -40,6 +36,9 @@ module.exports = class Router extends Backbone.Router
     the submitted form.
     ###
     auth: (path, options) ->
+        AuthView  = require '../views/auth'
+        AuthModel = require '../states/auth'
```

Comment:
I split the code concerning the Registration (onboarding) part below, and mode a lazy-require. So I put the _Auth_ requires here to be consistent, but it's trully only cosmetic :smile: